rootfs-builder/fedora/rootfs_lib.sh

```diff
@@ -70,7 +66,7 @@ build_rootfs()
 		die "neither yum nor dnf is installed"
 	fi
 
-	info "Using : ${PKG_MANAGER} to pull packages from ${REPO_URL}"
+	#info "Using : ${PKG_MANAGER} to pull packages."
```

Comment:
Can this line be removed entirely if it's no longer needed?

Reply:
@jodh-intel done.